### PR TITLE
ydiff 1.2

### PR DESCRIPTION
**Pull request by sayomaki**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by willi123yao** [comment]
This formula is still using python2, which is deprecated since 1/1/20. It should be able to work on python3, I will try to update the formula to build and run using python3.

File: Formula/ydiff.rb [hunk 1]
```diff
@@ -1,14 +1,18 @@
 class Ydiff < Formula
   desc "View colored diff with side by side and auto pager support"
   homepage "https://github.com/ymattw/ydiff"
-  url "https://github.com/ymattw/ydiff/archive/1.1.tar.gz"
-  sha256 "4c749204d1368557e74392845de22155201484edb2fdb5f97730cd6282c2dcca"
+  url "https://github.com/ymattw/ydiff/archive/1.2.tar.gz"
+  sha256 "0a0acf326b1471b257f51d63136f3534a41c0f9a405a1bbbd410457cebfdd6a1"
 
   bottle :unneeded
```
Comment (chenrui333) [inline Formula/ydiff.rb:7]:
```suggestion
```

Comment (chenrui333) [reply Formula/ydiff.rb:7]: maybe, I am wrong about this.

**Review by chenrui333 — APPROVED** [review]
Thanks!

**Comment by chenrui333** [comment]
I will follow up to do the license update.

**Pull request merged**